### ISSUE TITLE
 yt-dlp Command Fails with Error 429 and Requires Sign-In

### ISSUE THREAD

**Issue by ominfowave** [yt-dlp/yt-dlp#10890]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

India

### Provide a description that is worded well enough to be understood

The yt-dlp command was working fine previously but is now failing to download YouTube videos with the following errors:

Command:
yt-dlp https://youtube.com/shorts/8OLAi6Eba98?si=JZFqupY1p9FBF86d

Errors:
WARNING: [youtube] Unable to download webpage: HTTP Error 429: Too Many Requests
ERROR: [youtube] 8OLAi6Eba98: Sign in to confirm you’re not a bot. This helps protect our community.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[youtube] Extracting URL: https://youtube.com/shorts/8OLAi6Eba98?si=JZFqupY1p9FBF86d
[youtube] 8OLAi6Eba98: Downloading webpage
WARNING: [youtube] Unable to download webpage: HTTP Error 429: Too Many Requests
[youtube] 8OLAi6Eba98: Downloading ios player API JSON
[youtube] 8OLAi6Eba98: Downloading iframe API JS
[youtube] 8OLAi6Eba98: Downloading player 57c75fa4
[youtube] 8OLAi6Eba98: Downloading web creator player API JSON
ERROR: [youtube] 8OLAi6Eba98: Sign in to confirm you’re not a bot. This helps protect our community. Learn more
```

**Comment by seproDev** [comment]
Duplicate of #10128
Your IP has been blocked by YouTube

**Comment by ominfowave** [comment]
> Duplicate of #10128 Your IP has been blocked by YouTube

Is the block permanent or temporary - any idea?

**Comment by bashonly** [comment]
you have not provided enough info (e.g. a verbose log) for anyone to answer that question

based on the 429 error from the webpage request, I would guess this is a data center IP that has been blacklisted

**Comment by ominfowave** [comment]
> you have not provided enough info (e.g. a verbose log) for anyone to answer that question
> 
> based on the 429 error from the webpage request, I would guess this is a data center IP that has been blacklisted

It looks like it was just a temporary block. It was back up and running again around midnight our time.
